### PR TITLE
chore(faq-plus): remove aad plugin files

### PR DESCRIPTION
**Pull request by dilin-MS**

### CODE REVIEW

File: faq-plus/.fx/configs/projectSettings.json [hunk 1]
```diff
@@ -8,15 +8,19 @@
...
+            "fx-resource-local-debug",
+            "fx-resource-appstudio"
+        ],
+        "azureResources": []
     },
     "isFromSample": true,
     "programmingLanguage": "typescript",
-    "version": "2.0.0"
+    "version": "2.0.0",
+    "pluginSettings": {
```
Comment (KennethBWSong) [inline faq-plus/.fx/configs/projectSettings.json:21]: Not sure whether this part is needed. May need to confirm with qianhao?

**Review by SLdragon — APPROVED** [review]

File: faq-plus/.fx/configs/projectSettings.json (hunk 1 above)
Comment (dilin-MS) [reply faq-plus/.fx/configs/projectSettings.json:21]: @swatDong for confirmation

Comment (swatDong) [reply faq-plus/.fx/configs/projectSettings.json:21]: This is optional for now, but I would like to suggest to use the latest schema, to save potential migration effort.
Follow the latest scaffolding template, better to use:

``` json
    "pluginSettings": {
        "fx-resource-bot": {
            "host-type": "app-service",
            "capabilities": []
        }
    }
```

Comment (dilin-MS) [reply faq-plus/.fx/configs/projectSettings.json:21]: Fixed.

**Review by swatDong — APPROVED** [review]

**Pull request merged**